source/runtime.cpp

```diff
@@ -960,10 +960,6 @@ void reshade::runtime::load_config()
 	else
 		_current_preset_path = g_reshade_dll_path.parent_path() / current_preset_path;
 
-#if RESHADE_GUI
-	_current_browse_path = _current_preset_path.parent_path();
-#endif
```

Comment:
This is required 😢
This code will make sense if the current preset is not same as the DLL folder.

Reply:
This wasn't in master before though... So must have gotten lost somewhere else previously already.